### PR TITLE
Implement front end update

### LINKED ISSUE #97: Integrate updated front-end and back-end changes between client and server

### PR DESCRIPTION
**Pull request by JadenBalogh**

### CODE REVIEW

**Comment by JadenBalogh** [comment]
Please do a review of the updated website prior to our meeting on Tuesday, Nov 24th.

File: client/debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1123/223212.896:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (Jaredhw) [inline client/debug.log:1]: Do we want to include the debug log in our Version Control? We might want to add it to git ignore.

File: client/src/about/About.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        `}
+        image={blueprintsImg}
+      />
+      <AboutBlockLeft
+        title='Contact Us'
+        desc={`
+          Phone: 111-222-3333
+          E-Mail: Contact@HomeTogether.ca
+          Address: 1234 Home Together Street, Vancouver, BC, Canada.
+        `}
```
Comment (Jaredhw) [inline client/src/about/About.js:43]: No line breaks after each section, leads to it being mushed together on the site. Consider adding `<br />` after each line.

File: client/src/about/AboutBlockLeft.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+function AboutBlockLeft(props) {
+  return (
+    <div className='about-block'>
```
Comment (Jaredhw) [inline client/src/about/AboutBlockLeft.js:5]: Good idea to standardize this!

File: client/src/index.css [hunk 4]
```diff
@@ -1,3 +1,5 @@
+/* https://coolors.co/cfe5f8-a8c9ec-7099d2-01161e-f56476 */
```
Comment (Jaredhw) [inline client/src/index.css:1]: At some point later we'll have to go through a color pallets with our client and check with the accessibility guidelines.
See: https://refactoringui.com/previews/building-your-color-palette/ & https://commandcenter.blogspot.com/2020/09/color-blindness-is-inaccurate-term.html & https://www.toptal.com/designers/colorfilter but I like the current design it looks nice.

**Review by Jaredhw — APPROVED** [review]
Everything looks good. Manually going through all the functionality everything worked correctly. Going forward we may want to implement something like a CSS pre-processor but that's very out of scope for this sprint. I also try to add more sample data for the listing so we at least have 1 entry for each category when I have time. 

Approved!

**Comment by a1v1n1** [comment]
Looks good and works good

**Review by aKrisnanto — APPROVED** [review]
Looks good

**Pull request merged**